skycoin/skywire
# Feature/default stcp listen

Fixes #71 

 Changes:	
-	add function to get local IP address

How to test this PR:

Build, install and run 

`skywire-cli node gen-config`. Check whether the field for local address is correctly set.
